test/coverlet.core.tests/CoverageResultTests.cs

```diff
@@ -0,0 +1,160 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Coverlet.Core;
+using Coverlet.Core.Enums;
+using Moq;
+using Xunit;
+
+namespace Coverlet.Core.Tests
+{
+    using ThresholdTypeFlagValues = Dictionary<ThresholdTypeFlags, double>;
+
+    public class CoverageResultTests
+    {
+        private Modules _modules;
+
+        public CoverageResultTests()
```

Comment:
please add some comment on expected percentage to make test more clear for instance
```
System.Void Coverlet.Core.Tests.CoverageResultTests::CoverageResultTests() -> xx% lines xx% branches xx% methods
```